<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_savelog">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存日志</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_refresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_clearlog">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>清除显示区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_filter">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>数据解析</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DataReceived">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>651</width>
      <height>791</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>1031</width>
      <height>771</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_plot_clear">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重新绘制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_pause">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停采样</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测温仪</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_refresh_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>8</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_open_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
